gsa/src/web/pages/reports/details/thresholdpanel.js

```diff
@@ -0,0 +1,168 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import React from 'react';
+
+import styled from 'styled-components';
+
+import _ from 'gmp/locale';
+
+import Divider from 'web/components/layout/divider';
+
+import InfoPanel from 'web/components/panel/infopanel';
+
+import FilterIcon from 'web/components/icon/filtericon';
+
+import PropTypes from 'web/utils/proptypes';
+
+import ReportPanel from './reportpanel';
+import EditIcon from 'web/components/icon/editicon';
+import FootNote from 'web/components/footnote/footnote';
+import Layout from 'web/components/layout/layout';
+
+const UpdatingDivider = styled(({isUpdating, ...props}) => (
+  <Divider {...props} />
+))`
+  opacity: ${props => (props.isUpdating ? '0.2' : '1.0')};
+`;
+
+const ThresholdPanel = ({
+  entityType,
+  filter,
+  isUpdating = false,
+  onFilterChanged,
+  onFilterEditClick,
+}) => {
+  const levels = filter.get('levels', '');
+  const severity = filter.get('severity', 0);
+
+  const handleRemoveLogLevel = () => {
+    if (levels.includes('g')) {
+      const newLevels = levels.replace('g', '');
+      const lfilter = filter.copy();
+      lfilter.set('levels', newLevels);
+
+      onFilterChanged(lfilter);
+    }
+  };
+
+  const handleRemoveLowLevel = () => {
+    if (levels.includes('l')) {
+      const newLevels = levels.replace('l', '');
+      const lfilter = filter.copy();
+      lfilter.set('levels', newLevels);
+
+      onFilterChanged(lfilter);
+    }
+  };
+
+  const handleRemoveMediumLevel = () => {
+    if (levels.includes('m')) {
+      const newLevels = levels.replace('m', '');
+      const lfilter = filter.copy();
+      lfilter.set('levels', newLevels);
+
+      onFilterChanged(lfilter);
+    }
+  };
+  const handleSetMinimumSeverity = () => {
+    const lfilter = filter.copy();
+    lfilter.set('severity', 7.0, '>');
+
+    onFilterChanged(lfilter);
+  };
+  return (
+    <UpdatingDivider
+      flex="column"
+      align={['start', 'stretch']}
+      grow
+      isUpdating={isUpdating}
+    >
+      <InfoPanel
+        heading={_(
+          "The {{entityType}} can't be displayed, because the " +
+            'report contains to many results. Please decrease the number ' +
+            'of results by applying a different filter.',
+          {entityType},
+        )}
+      />
+      <Divider wrap>
+        {levels.includes('g') && (
+          <ReportPanel
+            icon={props => <FilterIcon {...props} />}
+            title={_('Log messages are currently included.')}
+            onClick={handleRemoveLogLevel}
+          >
+            {_('Exclude log messages in your filter settings.')}
```

Comment:
This sounds like the filter settings will not consider the keyword for log messages as such, instead of removing them from the results by filtering them out.
```suggestion
            {_('Filter out log messages via your filter settings.')}
```